addons/index.md

```diff
@@ -8,15 +8,67 @@ layout: documentation
-Add-ons can be easily installed through the Paper UI (menu entry "Add-ons").
-
-
-| Add-on Type | Description |
-|-------------|----------------------|
-| [Bindings](bindings.html) | Bindings integrate physical hardware, external systems and web services in openHAB |
-| [User Interfaces](uis.html) | User interfaces are either native smartphone apps or web applications that access the openHAB server through the REST API |
-| [Persistence](persistence.html) | Persistence services allow openHAB to store time series data for history-based actions or statistics |
-| [Actions](actions.html) | Actions are predefined methods for openHAB rules and scripts |
-| [Transformations](transformations.html) | Transformations are used to translate between technical and human-readable values for items |
-| [Voice Services](voices.html) | Services that provide voice enabling features, such as text-to-speech, speech-to-text etc. |
-| [3rd Party System Integration](io.html) | Expose openHAB to external systems |
+There are several ways you can install an add-on. 
+These are described under *Installation of Add-ons* below
+
+| Add-on Type                             | Description                                                                                                               |
+|-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------|
+| [Bindings](bindings.html)               | Bindings integrate physical hardware, external systems and web services in openHAB                                        |
+| [User Interfaces](uis.html)             | User interfaces are either native smartphone apps or web applications that access the openHAB server through the REST API |
+| [Persistence](persistence.html)         | Persistence services allow openHAB to store time series data for history-based actions or statistics                      |
+| [Actions](actions.html)                 | Actions are predefined methods for openHAB rules and scripts                                                              |
+| [Transformations](transformations.html) | Transformations are used to translate between technical and human-readable values for items                               |
+| [Voice Services](voices.html)           | Services that provide voice enabling features, such as text-to-speech, speech-to-text etc.                                |
+| [3rd Party System Integration](io.html) | Expose openHAB to external systems                                                                                        |
+
+## Installation of Add-ons
+
+Depending on the [package]({{base}}/configuration/packages.html) you have choosen during your first time setup, there are already some pre-installed add-ons.
+Additional add-ons can be installed in the different ways, described below.
```

Comment:
That's missing completely and it seems 3 people didn't recognize that yet.

I have never used that option.

Reply:
Ah right. I also didn't realize that 🙈

Reply:
To be honest.
Since i didn't use this option, can someone point me to a location (e.g. in the forum) where i can read through something?

Transfer knowledge is the following from what i know and read:
I have to know the binding id.
(Thankfully i have added a naming convention section to the bundle management article some time ago.)

Is it enough to just edit the addons.cfg from
```binding=astro,fritzboxtr0641```
to (for example):
```binding=astro,fritzboxtr0641,network```
to install the network binding?
Is there somethign else to pay attention too?

I have tested it and network binding was installed, so that sounds logic.
I just don't want to miss somethign since i didn't use that file yet.

Reply:
That is correct. The relevant part of the `addons.cfg` needs to be uncommented first. What is not mentioned in the file: openHAB 1.x bindings need to be named with a trailing "1". See: https://community.openhab.org/t/upgrade-has-killed-my-install-desperate-times/38293/9
An easy way to find the correct name may be to check the docs page binding URL, e.g.: https://docs.openhab.org/addons/bindings/mqtt1/readme.html

Reply:
@kaikreuzer should we mention the "1" here? https://github.com/openhab/openhab-distro/blob/master/distributions/openhab/src/main/resources/conf/services/addons.cfgf

Reply:
I have adressed both topics @ThomDietrich

Reply:
> openHAB 1.x bindings need to be named with a trailing "1".

FTR, as I understand it, that only applies to bindings specified in some sections of `addons.cfg`. For other sections, such as actions and persistence, the `1` is not required.

For example, from my addons.cfg, note that gcal persistence doesn't have a `1`, while gcal io (in misc section) has a `1`.
```
# A comma-separated list of persistence services to install (e.g. "rrd4j,jpa")
persistence = mysql,rrd4j,gcal

# A comma-separated list of actions to install (e.g. "mail,pushover")
action = mail,ecobee

# A comma-separated list of transformation services to install (e.g. "map,jsonpath")
transformation = javascript,jsonpath,map

# A comma-separated list of voice services to install (e.g. "marytts,freetts")
voice = voicerss

# A comma-separated list of miscellaneous services to install (e.g. "myopenhab")
misc = gcal1,market,restdocs
```

Reply:
@kaikreuzer is the trailing `1` definetely only for the binding area or are others affected as well?

Reply:
Yes, see https://github.com/openhab/openhab1-addons/blob/master/features/openhab-addons/src/main/feature/feature.xml

Reply:
Ok. I will point that out and think we are good to merge then afterwards for this initial document.

Reply:
Looking @ feature xml the `misc` area has also trailing `1`'s.
So i will add that too.